nautobot/ipam/views.py

```diff
@@ -1161,75 +1161,32 @@ def list(self, request, *args, **kwargs):
 #
 # VLAN groups
 #
-
-
-class VLANGroupListView(generic.ObjectListView):
-    queryset = VLANGroup.objects.all()
-    filterset = filters.VLANGroupFilterSet
-    filterset_form = forms.VLANGroupFilterForm
-    table = tables.VLANGroupTable
-
-
-class VLANGroupView(generic.ObjectView):
+class VLANGroupUIViewSet(NautobotUIViewSet):
+    bulk_update_form_class = forms.VLANGroupBulkEditForm
+    filterset_class = filters.VLANGroupFilterSet
+    filterset_form_class = forms.VLANGroupFilterForm
+    form_class = forms.VLANGroupForm
     queryset = VLANGroup.objects.all()
+    serializer_class = serializers.VLANGroupSerializer
+    table_class = tables.VLANGroupTable
 
-    def get_extra_context(self, request, instance):
-        vlans = (
-            VLAN.objects.restrict(request.user, "view")
-            .filter(vlan_group=instance)
-            .prefetch_related(Prefetch("prefixes", queryset=Prefix.objects.restrict(request.user)))
-        )
-        vlans_count = vlans.count()
-
-        data_transform_callback = get_add_available_vlans_callback(show_available=True, vlan_group=instance)
```

Comment:
Need to preserve this functionality in the vlans table (augmenting the `vlan_group.vlans` queryset with "available" not-yet-allocated vlans).